src/modules/px4iofirmware/mixer.cpp

```diff
@@ -513,6 +518,11 @@ mixer_set_failsafe()
 	float	outputs[PX4IO_SERVO_COUNT];
 	unsigned mixed;
 
+	// update slew rate value
+	float slew_max = 2.0f * 1000.0f * dt / (r_page_servo_control_max[0] - r_page_servo_control_min[0]) / REG_TO_FLOAT(
```

Comment:
And again here. Or think about making a define for `2.0f` and `1000.0f` which explains them and puts it in a single place.